tests/modules/nf-core/parabricks/fq2bam/nextflow.config

```diff
@@ -0,0 +1,9 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+    memory = "30 GB"
+    cpus  = 8
+    accelerator = 1
+}
+docker.runOptions = "--gpus all"
+singularity.runOptions = "--nv"
```

Comment:
We should add that in an extra profile in both this repo and the template.
But it can stay here for now

Reply:
Are there other tools in here that depend on GPUs that we can use as an example?

Reply:
I think we use to have guppy, but not sure about the status of this tool anymore...